src/pages/Options.tsx

```diff
@@ -44,7 +44,11 @@ export const DEFAULT_TOOL_CONFIG: ToolConfig = {
   },
 };
 export const MULTIMODAL_MODELS = ["llava", "bakllava"];
-export const EMBEDDING_MODELS = ["nomic-embed-text", "all-minilm"];
+export const EMBEDDING_MODELS = [
+  "nomic-embed-text",
+  "all-minilm",
```

Comment:
I recall looking a while back (time flies) ot Ollama code to see if there was a way to list models only of a certain type, but now I can't remember what I found!

Reply:
I'm not aware of any functionality in the [Ollama API](https://github.com/ollama/ollama/blob/main/docs/api.md). Although, I agree that this would be a useful feature.